app/assets/stylesheets/main.sass

```diff
@@ -77,3 +77,84 @@ form
 .visitor-tip
   display: flex
   justify-content: center
+
+
+#background
+  background:
+    image: asset-url("welcome.jpg")
+    size: cover
+    repeat: no-repeat
+    position: 50% 50%
+  display: flex
+  justify-content: space-around
+  font-family: 'Tajawal', cursive
+  height: 100vh
+
+  #logo-links
+    display: flex
+    justify-content: center
+    align-self: center
+    flex-flow: column nowrap
+    background-color: transparentize(#AAAAAA, .1)
+    position: relative
+    width: auto
+    height: auto
+    border-radius: 10px
+
+  #logo
+    flex: 2
+    display: flex
+    justify-content: center
+    align-items: center
+    flex-direction: column
+    padding: 20px 20px 0px 20px
+    height: 400px
+
+  img
+    height: 150px
+    width: 150px
+
+  h3
+    margin-top: 20px
+    font-size: 70px
+    color: white
+
+#links
+  flex: 2
+  display: flex
+  justify-content: space-around
+  align-items: center
+  text-align: center
+  flex-flow: row nowrap
+
+  a
+    color: white
+    font-size: 35px
+
+#link1
+  flex: 1
+  display: flex
+  justify-content: center
+  align-items: center
+  background-color: transparentize(#DE6E4B, .4)
+  height: 100%
+  border-radius: 0px 0px 10px 10px
+  padding-top: 10px
+
+#link2
```

Comment:
I noticed these aren't actually being used. What are they for?

Reply:
Correct...very observant! I left those in there in case we want to add more links. If we choose not to, we can delete #link2, and #link3 without any impact.